test/bigtest/interactors/EditVoucherView.js

```diff
@@ -0,0 +1,19 @@
+import {
+  interactor,
+  isPresent,
+} from '@bigtest/interactor';
+
+export default interactor(class EditVoucherView {

```

Comment:
`EditVoucherView.js` looks like mixed-up. usually it's `VoucherEditInteractor` or something